docs/website/docs/general-usage/schema-evolution.md

```diff
@@ -0,0 +1,205 @@
+    # Column added:
+    "CEO": "Alice Smith", 
+    "address": {
+        # 'building' renamed to 'main_block'
+        'main_block': 'r&d', 
+	      # Removed room column
+        # "room": 7890,  
+    },
+    "Inventory": [
+        # Type change: 'inventory_nr' changed to string from int
+        {"name": "Plasma ray", "inventory nr": "AR2411"}, 
+        {"name": "Self-aware Roomba", "inventory nr": "AR268"},  
+        {"name": "Type-inferrer", "inventory nr": "AR3621"}  
+    ]
+}
+```
+
+Let’s load the data and look at the tables:
+<iframe width="560" height="315" src='https://dbdiagram.io/e/65e80303cd45b569fba28e9d/65e80556cd45b569fba2b8ab'> </iframe>
+
+What happened?
+
+- Added column:
+    - A new column named “ceo” is added  to the “org” table.
+- Variant column:
+    - A new column named `inventory_nr__v_text` is added as the datatype of the column was changed from “integer” to “string”.
+- Removed column stopped loading:
+    - New data to column “room” is not loaded.
+- Column stopped loading and new one was added:
+    - A new column “address__building” was added and now data will be loaded to that and stop loading in the column “address__main_block”.
+
+## Alert schema changes to curate new data
+
+By separating the technical process of loading data from curation, you free the data engineer to do engineering, and the analytics to curate data without technical obstacles. So, the analyst must be kept in the loop.
+
+**Keeping track of column lineage**
+
+By loading the ‘load_info’ to the destination,  info about the column ‘data types’, ‘add times’ and ‘load id’. To read more please see [the data lineage article](https://dlthub.com/docs/blog/dlt-data-lineage) we have on the blog.
+
+**Getting notifications**
+We can read the load outcome and send it to slack webhook with dlt.
+```python
+# Import the send_slack_message function from the dlt library
+from dlt.common.runtime.slack import send_slack_message
+
+# Define the URL for your Slack webhook
+hook = "https://hooks.slack.com/services/xxx/xxx/xxx"
+
+# Iterate over each package in the load_info object
+for package in info.load_packages:
+    # Iterate over each table in the schema_update of the current package
+    for table_name, table in package.schema_update.items():
+        # Iterate over each column in the current table
+        for column_name, column in table["columns"].items():
+            # Send a message to the Slack channel with the table 
+						# and column update information
+            send_slack_message(
+                hook,
+                message=(
+                    f"\tTable updated: {table_name}: "
+                    f"Column changed: {column_name}: "
+                    f"{column['data_type']}"
+                )
+            )
+```
+This script sends Slack notifications for data schema updates using the 'send_slack_message' function from the dlt library. It provides details on the updated table and column.
```

Comment:
Remove data.

This script sends Slack notifications for schema updates

Reply:
use code formatting for code, for example function names: `send_slack_message`

Reply:
done